build/build.xml

```diff
@@ -844,7 +837,36 @@
       <env key="APPCONFIG_DATA_DIRECTORY" value="${aspace.data_directory.test}" />
       <env key="APPCONFIG_DB_URL" value="${env.ASPACE_TEST_DB_URL}" />
       <env key="APPCONFIG_SOLR_URL" value="${env.ASPACE_TEST_SOLR_URL}" />
-      <arg line="../build/${jruby_build_bin_path}/rspec -b --format d --order rand:1 ${example-arg} spec/${spec-arg}" />
+      <arg line="../build/${jruby_build_bin_path}/rspec -b --format d --tag ~db --order rand:1 ${example-arg} spec/${spec-arg}" />
```

Comment:
@quoideneuf maybe the tag could be a parameter?

```xml
  <target name="public:test" depends="set-classpath, public:copy-shared-resources, setup-integration" description="Run the unit test suite">
    <property name="tag" value="~db"/>
    <!-- ... -->
    <arg line="../build/${jruby_build_bin_path}/rspec -b --format d --tag ${tag} --order rand:1 ${example-arg} spec/${spec-arg}" />
```

So `public:test` would work the same, and could use `public:test -Dtag="db:accessibility"` to target those (and the specific task can be removed)?

Reply:
@mark-cooper seems like a good idea to me, but I don't think it can be done with the current ant task dependencies unless we get into ant add ons that provide conditionals, because the `accessibility` path needs `nuke:test` not to happen. But I think it can be done by pulling some of the dependencies out of `build.xml` and making the GitHub workflows more verbose. 

```
    - name: Run Public tests
      env:
        ASPACE_TEST_DB_URL: jdbc:mysql://127.0.0.1:${{ job.services.db.ports[3306] }}/archivesspace?useUnicode=true&characterEncoding=UTF-8&user=as&password=as123&useSSL=false&allowPublicKeyRetrieval=true
      run: |
        ./build/run setup:integration
        ./build/run public:test

    - name: Run Public accessibility tests
      env:
        ASPACE_TEST_DB_URL: jdbc:mysql://127.0.0.1:${{ job.services.db-accessibility.ports[3306] }}/accessibility?useUnicode=true&characterEncoding=UTF-8&user=as&password=as123&useSSL=false&allowPublicKeyRetrieval=true
      run: |
        ./build/run db:migrate:test
        ./build/run solr:reset:test
        ./build/run public:test -Dtag=accessibility
```
This seems like the right way to go to me (at the moment at least), and more generally I wonder if moving build logic out of ant and into workflows won't going to make life easier and help us move away from ant.